packages/curriculum-compiler-json/compilers/helpers.js

```diff
@@ -0,0 +1,33 @@
+const { getCompiler } = require('@enkidevs/curriculum-compiler-string')
+const { contentTypes } = require('@enkidevs/curriculum-helpers')
+
+function getInsightCompiler () {
+  return getCompiler(contentTypes.INSIGHT)
+}
+function compileNodeToInsightMarkdown (node) {
+  if (!node || !node.children) {
+    throw new Error('Cannot compile invalid node')
+  }
+  return getInsightCompiler().compileSync({
+    type: 'root',
+    children: node.children
+  })
+}
+
+function getAnswersFromNode (node) {
+  const answersASTList = node.children.find(
+    child => child.type === 'list' && child.answers
+  )
+  if (!answersASTList) {
+    throw new Error('No answers in question node')
+  }
+  return compileNodeToInsightMarkdown({ children: [answersASTList] })
+    .split('\n')
+    .filter(Boolean)
+    .map(i => i.substring(1).trim())
```

Comment:
So this would contain `*x*` for a list like this?

```md
* *x*
* *b*
```

Reply:
Should we perhaps make each answer an object that contains the text and the `correct` flag?

Reply:
1) yeah exactly 
2) the thing is we can have multiple correct answers, ordered if there are multiple `???` gaps. we'd need a way of expressing that as well